Core/Utilities/Proxy.py

```diff
@@ -104,30 +103,32 @@ def wrapped_fcn(*args, **kwargs):
   return wrapped_fcn
 
 
-def getProxy(userDNs, userGroup, vomsAttr, proxyFilePath):
-  """ do the actual download of the proxy, trying the different DNs
+def getProxy(user, userGroup, vomsAttr, proxyFilePath):
```

Comment:
Here you are changing the signature of the function. It should still be OK but please evaluate it carefully.

Reply:
OK